app/controllers/application_controller.rb

```diff
@@ -10,7 +10,17 @@ class ApplicationController < ActionController::Base
   after_action :verify_authorized, :except => :index, unless: :devise_controller?
   #after_action :verify_policy_scoped, :only => :index, unless: :devise_controller?
 
+  before_action :set_locale
 
+  def set_locale
+    if cookies[:placeme_locale] && I18n.available_locales.include?(cookies[:placeme_locale].to_sym)
```

Comment:
What do you think in adding a new attribute to the User model to store the User preferred locale instead of using cookies??
So you can do something like: 
```ruby
if current_user.locale &&I18n.available_locales.include?(current_user.locale.to_sym)
...
````

This has the advantage that if you change the device where you are using the app you are going to have your locale always set up correctly.

Also I'm not a big fan of using variables named with just one letter. I think is more readable calling the variable `locale `  instead of `l`